subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -544,6 +551,23 @@ public AbstractTestTask setTestNameIncludePatterns(List<String> testNamePattern)
         return this;
     }
 
+    @Option(option = "fail-fast", description = "Stops test execution after the first failed test")
+    public AbstractTestTask setFailFast(boolean ignored) {
```

Comment:
The ignored parameters are somewhat unintuitive when reading the methods but I guess there's no way around it as you want to have `--no-fail-fast` as well. Some comment above: I am wondering if we actually need `--no-fail-fast`.

Reply:
done